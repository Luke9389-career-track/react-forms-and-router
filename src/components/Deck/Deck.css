.Deck{
  margin-left: 20px;
  display: flex;
  overflow: scroll;
  border-top: 2px black solid;
  border-bottom: 2px black solid;
  background: linear-gradient(90deg, rgba(255,255,255,1) 73%, rgb(140, 140, 140) 100%);
}

.arrow{
  font-size: 50px;
  margin-right: 20px;
  display: block;
  float: right;
}
