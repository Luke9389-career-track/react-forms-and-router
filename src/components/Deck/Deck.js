import React from 'react';
import PropTypes from 'prop-types';

function Deck({ title, children }) {

  return (
    <section>
      <header>
        <h2>{title}</h2>
      </header>
      {children}
    </section>
  );
}

Deck.propTypes = {
  title: PropTypes.string.isRequired,
  children: PropTypes.node.isRequired
};

export default Deck;
