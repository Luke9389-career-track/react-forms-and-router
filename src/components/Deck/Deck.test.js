import React from 'react';
import { shallow } from 'enzyme';
import Deck from './Deck';

describe('Deck component', () => {
  it('renders Deck', () => {
    const wrapper = shallow(<Deck title='Test'>
      <p key={1}></p>
      <p key={2}></p>
    </Deck>);
    expect(wrapper).toMatchSnapshot();
  });
});
