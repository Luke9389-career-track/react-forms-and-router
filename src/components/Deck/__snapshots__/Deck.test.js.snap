// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Deck component renders Deck 1`] = `
<section>
  <header>
    <h2>
      Test
    </h2>
  </header>
  <p
    key="1"
  />
  <p
    key="2"
  />
</section>
`;
